ansible/roles/qemu/tasks/main.yml

```diff
@@ -0,0 +1,123 @@
+---
+- name: install packages (apt)
+  apt:
+    name: '{{ item.name }}'
+    state: '{{ item.state }}'
+    update_cache: true
+    force_apt_get: true
+    dpkg_options: force-confnew
+  with_items:
+  - { name: qemu, state: latest }
+  - { name: qemu-kvm, state: latest }
+  - { name: qemu-utils, state: latest }
+  - { name: uml-utilities, state: latest }
+  - { name: virt-manager, state: latest }
+  - { name: dmg2img, state: latest }
+  - { name: libguestfs-tools, state: latest }
+  - { name: bridge-utils, state: latest }
+  - { name: python3-lxml, state: latest }
+  - { name: libosinfo-bin, state: latest }
+  environment:
+    DEBIAN_FRONTEND: noninteractive
+  when: ansible_os_family == 'Debian' or ansible_os_family == 'Ubuntu'
+  tags: qemu
+
+- name: forward bridge traffic
```

Comment:
Is this persistent?

Reply:
Are packages installed with `apt` persist on a system? Yes. Until you destroy it, I guess?